src/main/java/frc/robot/util/MoPrefs.java

```diff
@@ -135,14 +139,23 @@ public class MoPrefs {
     public static final UnitPref<Time> intakeCurrentSenseTime =
             secondsPref("Intake Current Sense Time", Units.Seconds.of(0.1));
     public static final UnitPref<Current> intakeCurrentSenseThreshold =
-            amperesPref("Intake Current Sense Threshold", Units.Amps.of(10));
+            ampsPref("Intake Current Sense Threshold", Units.Amps.of(10));
+
+    public static final Pref<Double> climberZeroPwr = unitlessDoublePref("Climber Zero Power", -0.3);
+    public static final UnitPref<Time> climberZeroTimeCutoff = secondsPref("Climber Zero Time", Units.Seconds.of(0.25));
+    public static final UnitPref<Current> climberZeroCurrentCutoff =
+            ampsPref("Climber Zero Current", Units.Amps.of(80));
+    public static final UnitPref<Velocity<Angle>> climberMotorSpeed =
+            rotationsPerSecPref("Climber Motor Speed", Units.RotationsPerSecond.of(75));
+    public static final UnitPref<Distance> climberMinimum = centimetersPref("Climber Minimum", Units.Centimeters.of(2));
```

Comment:
The preference `climberMinimum` isn't used anywhere that I can see.

There are two approaches to ensuring that the climber doesn't actually run against the hard stops when in normal operation.
1. When zeroing, you actually set the encoder to some negative value. Then you set the soft limit to zero.
2. When zeroing, you set the encoder to zero. Then, you set the soft limit to something greater than zero.

Either approach will work. We've been using 1. on all the other subsystems, but feel free to use whichever you prefer.

But do implement one of these approaches. Currently you're both setting the encoder to zero when zeroing, and you're setting the soft limit to zero. This leaves no margin for error if the motor should happen drive a bit past the soft limits between iterations of the control loop. Better to use one of these approaches to add some buffer, so the mechanism never actually contacts the limits (except when zeroing).